gensim/models/fasttext.py

```diff
@@ -955,6 +950,15 @@ def load(cls, *args, **kwargs):
                 model.trainables.vectors_vocab_lockf = ones(len(model.trainables.vectors), dtype=REAL)
             if not hasattr(model.trainables, 'vectors_ngrams_lockf') and hasattr(model.wv, 'vectors_ngrams'):
                 model.trainables.vectors_ngrams_lockf = ones(len(model.trainables.vectors), dtype=REAL)
+
+            if not hasattr(model, 'compatible_hash'):
+                logger.warning(
+                    "this older model was trained with a buggy hash function, ",
+                    "please consider retraining"
```

Comment:
but this doesn't help, because hash will be still broken

Reply:
OK, I improved the warning